### PR TITLE
Packages vs packages

### PR DESCRIPTION
**Pull request by bpulliam**
The Nuget.config was creating a packages directory with capital 'P', but the project files and the build scripts referred to a lower-case 'p'. This standardizes on lower-case.
It also fixes the templates and recent project additions to use the NugetPackageDirectory variable from Directory.Build.props to eliminate repetition.

### CODE REVIEW

**Review by DrusTheAxe — APPROVED** [review]

File: dev/Kozani/KozaniAppGraph/KozaniAppGraph.vcxproj [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  <Import Project="$(RepoRoot)\packages\Microsoft.Build.Tasks.Git.$(MicrosoftBuildTasksGitVersion)\build\Microsoft.Build.Tasks.Git.props" Condition="Exists('$(RepoRoot)\packages\Microsoft.Build.Tasks.Git.$(MicrosoftBuildTasksGitVersion)\build\Microsoft.Build.Tasks.Git.props')" />
-  <Import Project="$(RepoRoot)\packages\Microsoft.SourceLink.Common.$(MicrosoftSourceLinkCommonVersion)\build\Microsoft.SourceLink.Common.props" Condition="Exists('$(RepoRoot)\packages\Microsoft.SourceLink.Common.$(MicrosoftSourceLinkCommonVersion)\build\Microsoft.SourceLink.Common.props')" />
-  <Import Project="$(RepoRoot)\packages\Microsoft.Windows.CppWinRT.$(MicrosoftWindowsCppWinRTVersion)\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(RepoRoot)\packages\Microsoft.Windows.CppWinRT.$(MicrosoftWindowsCppWinRTVersion)\build\native\Microsoft.Windows.CppWinRT.props')" />
+  <Import Project="$(NugetPackageDirectory)\Microsoft.SourceLink.GitHub.$(MicrosoftSourceLinkGitHubVersion)\build\Microsoft.SourceLink.GitHub.props" Condition="Exists('$(NugetPackageDirectory)\Microsoft.SourceLink.GitHub.$(MicrosoftSourceLinkGitHubVersion)\build\Microsoft.SourceLink.GitHub.props')" />
```
[the remark this answers is not in the document]
Comment (bpulliam) [reply dev/Kozani/KozaniAppGraph/KozaniAppGraph.vcxproj:4]: The top-level Directory.Build.props.

**Pull request merged**